keyboards/chlx/piche60/keyboard.json

```
@@ -0,0 +1,125 @@
+{
+    "manufacturer": "chlx bsmt",
+    "keyboard_name": "piche60",
+    "maintainer": "gaclee4b",
+    "bootloader": "atmel-dfu",
```

Bootmagic is not configured to use top left key
```suggestion
    "bootloader": "atmel-dfu",
    "bootmagic": {
        "matrix": [1, 1]
    },
```